cmd/forwarder/http.go

```diff
@@ -224,6 +224,14 @@ func (s *httpServer) process(req *http.Request, r io.Reader, remoteAddr string,
 		s.pMetadataLogsReceived.Inc(numLogs)
 	}
 
+	// Scrub tokens from fixedBody
+	for user, token := range s.Config.TokenMap() {
```

Comment:
As written, this puts the work done in TokenMap in the hot path of processing every request. Consider making TokenMap only do its work once and return a cached value or have somewhere else along the way call it once and store the value.

Reply:
@danp That's a really good catch, and easy fix, thanks!

Reply:
@danp I memoized `c.TokenMap()`, so it should generate on first call and then use a cached value going forward.